packs/slack/config.yaml

```diff
@@ -1,4 +1,26 @@
 ---
+# Used for auth test action
+auth_test_action:
+  token: ""
+
+# Used for list channels
+list_channels_action:
```

Comment:
Edit: I see user can also specify it as a parameter and those values there work more as a default value. I think I'm fine with that.